src/components/AddForm/AddForm.tsx

```diff
@@ -41,12 +49,11 @@ export const AddForm = ({ onSubmit }) => {
         required
       />
 
-      <input
-        type="submit"
-        className={operation.button}
-        data-testid={testIds.button}
+      <Button
+        onClick={() => handleSubmit}
```

Comment:
this one will return function instead of calling it
```suggestion
        onClick={handleSubmit}
```